src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Color {

```

Comment:
The enum for colors was defined as required, but it would be better to add more colors to make the lottery more interesting and random.

Reply:
Done it. But to be honest I feel that it changed absolutely nothing ;)